tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckConsistentCasing.java

```diff
@@ -64,19 +64,19 @@ public CheckCLDR handleSetCldrFileToCheck(
             } catch (Exception e) {
                 types = Collections.emptyMap();
             }
+            if ((types == null || types.isEmpty()) && !SpecialLocales.isScratchLocale(locale)) {
+                possibleErrors.add(
+                        new CheckStatus()
+                                .setCause(this)
+                                .setMainType(CheckStatus.warningType)
+                                .setSubtype(Subtype.incorrectCasing)
+                                .setMessage("Could not load casing info for {0}", locale));
+            }
         } else {
             // no casing info - since the types Map is global, and null checks aren't done,
             // we are better off  with an empty map here
             types = Collections.emptyMap();
         }
-        if ((types == null || types.isEmpty()) && !SpecialLocales.isScratchLocale(locale)) {
-            possibleErrors.add(
-                    new CheckStatus()
-                            .setCause(this)
-                            .setMainType(CheckStatus.warningType)
-                            .setSubtype(Subtype.incorrectCasing)
-                            .setMessage("Could not load casing info for {0}", locale));
-        }
```

Comment:
Previously, this warning would fire for Arabic, etc. because the script is not cased, so control would reach line 70.  Then types is checked again on this line and the warning fires.  

The change moves the warning to line 67, so it fires only if casing is expected but not present.